### PR TITLE
Breakfix/miniziti refactor issues

### PR DESCRIPTION
**Pull request by sabedevops**

### CODE REVIEW

**Comment by sabedevops** [comment]
@dovholuknf && @qrkourier This restores miniziti functionality. Can we please get this merged? There are a few, minor outstanding issues I will address through this branch later, but for now, this gets things back to sanity and working.

File: docusaurus/docs/learn/quickstarts/network/miniziti.bash [hunk 1]
```diff
@@ -471,7 +473,7 @@ main(){
 
         if check_command ziti &>/dev/null; then
             logWarn "Removing $MINIKUBE_PROFILE profile identity from ziti-cli.json"
-            ziti edge logout --cli-identity "$MINIKUBE_PROFILE" >&3
+            #ziti edge logout --cli-identity "$MINIKUBE_PROFILE" >&3
```
Comment (qrkourier) [inline docusaurus/docs/learn/quickstarts/network/miniziti.bash]: If this is intentionally commented, then why not also remove the enclosing check and log message?

**Review by qrkourier — CHANGES_REQUESTED** [review]
Should we double-check the output token files to ensure they resemble a JWT? At a minimum, a `test -s`, but perhaps a minimum byte count?

**Comment by sabedevops** [comment]
> Should we double-check the output token files to ensure they resemble a JWT? At a minimum, a `test -s`, but perhaps a minimum byte count?

I've added `--exit-status` to `jq` instead.

**Review by qrkourier — APPROVED** [review]

**Pull request merged**